modules/server/src/test/scala/observe/server/tcs/GuideConfigDbSuite.scala

```diff
@@ -151,12 +151,9 @@ final class GuideConfigDbSuite extends munit.FunSuite {
   }
```

Comment:
Weird that you can still have `shouldBe` under `munit`

Reply:
I missed that one, because it compiled nevertheless (because of an exiting import)